This file is part of MXE.
See index.html for further information.
This patch has been taken from:

https://github.com/rkanavath/ossim/commit/643f0b1f6984b97e7a72af7ff05085b7e716f934
This patch MUST be removed when building ossim version > 1.8.20
Merged in https://github.com/ossimlabs/ossim/pull/3

diff -burN ossim-1.8.20-1.orig/ossim/include/ossim/point_cloud/ossimGenericPointCloudHandler.h ossim-1.8.20-1/ossim/include/ossim/point_cloud/ossimGenericPointCloudHandler.h
--- ossim-1.8.20-1.orig/ossim/include/ossim/point_cloud/ossimGenericPointCloudHandler.h	2015-07-21 15:23:00.000000000 +0200
+++ ossim-1.8.20-1/ossim/include/ossim/point_cloud/ossimGenericPointCloudHandler.h	2015-12-04 12:29:00.001900000 +0100
@@ -75,7 +75,7 @@
    virtual void close() {  }

 protected:
-   ossimGenericPointCloudHandler() {}
+   ossimGenericPointCloudHandler();
    ossimPointBlock m_pointBlock;
 };

diff -burN ossim-1.8.20-1.orig/ossim/src/ossim/point_cloud/ossimGenericPointCloudHandler.cpp ossim-1.8.20-1/ossim/src/ossim/point_cloud/ossimGenericPointCloudHandler.cpp
--- ossim-1.8.20-1.orig/ossim/src/ossim/point_cloud/ossimGenericPointCloudHandler.cpp	1970-01-01 01:00:00.000000000 +0100
+++ ossim-1.8.20-1/ossim/src/ossim/point_cloud/ossimGenericPointCloudHandler.cpp	2015-12-04 12:27:33.000145000 +0100
@@ -0,0 +1,6 @@
+#include <ossim/point_cloud/ossimGenericPointCloudHandler.h>
+
+ossimGenericPointCloudHandler::ossimGenericPointCloudHandler()
+{
+
+}
